include/recognition/planar/HopcroftTarjan.hpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * IPlanarGraphRecognition.hpp
+ *
+ *  Created on: 24.03.2024
+ *      Author: Dzianis Lahunou
+ *      Ported by: Krzysztof Turowski (krzysztof.szymon.turowski@gmail.com)
+ */
+
+#pragma once
+
+#include "recognition/planar/IPlanarGraphRecognition.hpp"
+
+namespace Koala {
+
+const int left = 1;
+const int right = 2;
```

Comment:
Please use UPPERCASE - and as with Block, consider moving to the respective cpp file